diff --git a/backend/src/core/proxy-utils/parsers/index.js b/backend/src/core/proxy-utils/parsers/index.js
index fa5a4f6f..4d6825ea 100644
--- a/backend/src/core/proxy-utils/parsers/index.js
+++ b/backend/src/core/proxy-utils/parsers/index.js
@@ -319,9 +319,9 @@ function URI_SS() {
                         'skip-cert-verify': ['1', 'true', 1, true].includes(
                             params['skip-cert-verify'],
                         ),
-                        mux: /^\d+$/.test(params.mux)
-                            ? parseInt(params.mux, 10)
-                            : undefined,
+                        mux: params.mux === 'false' || params.mux === false ? false :
+                            params.mux === 'true' || params.mux === true ? true :
+                                isNaN(params.mux) ? params.mux : Number(params.mux),
                     };
                     break;
                 case 'shadow-tls': {
