diff --git a/backend/src/core/proxy-utils/parsers/index.js b/backend/src/core/proxy-utils/parsers/index.js
index 79f1601..610d02a 100644
--- a/backend/src/core/proxy-utils/parsers/index.js
+++ b/backend/src/core/proxy-utils/parsers/index.js
@@ -314,6 +314,7 @@ function URI_SS() {
                             getIfNotBlank(params['host']),
                         path: getIfNotBlank(params.path),
                         tls: getIfPresent(params.tls),
+                        mux: getIfPresent(Number(params.mux)),
                         // mux 目前有问题 v2rayN 有 bug...开了 mux 导出时也是 0, 如果改成 1 导入它就报错
                         // 有需求的自己用脚本操作吧
                     };
diff --git a/backend/src/core/proxy-utils/producers/sing-box.js b/backend/src/core/proxy-utils/producers/sing-box.js
index 0587ef7..3bfd39c 100644
--- a/backend/src/core/proxy-utils/producers/sing-box.js
+++ b/backend/src/core/proxy-utils/producers/sing-box.js
@@ -540,6 +540,7 @@ const ssParser = (proxy = {}) => {
                     case 'mux':
                         if (proxy['plugin-opts'].mux)
                             parsedProxy.multiplex = { enabled: true };
+                        optArr.push(`mux=${proxy['plugin-opts'].mux ?? true}`);
                         break;
                     default:
                         optArr.push(`${k}=${proxy['plugin-opts'][k]}`);
